packages/presentation/lib/di/injector_presentation.dart

```diff
@@ -44,6 +63,12 @@ void _initModuleApp() {
   );
 }
 
+void _initFirebaseAnalytics() {
+  GetIt.instance.registerSingleton<Analytics>(
+    Analytics(FirebaseAnalytics.instance),
+  );
+}
```

Comment:
Analytics is a service, which consumes events and sends them to the server under the hood. so, there's no difference with any other api service. so, where should we put it?)